content/index.fr.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Vue d'ensemble des standards de l'accessibilité"
+permalink: /fundamentals/fr
+ref: /fundamentals/
+lang: fr
+last_updated: 2020-12-28
+translators:
+- name : "Sofia Ahmed"
+contributors:
+- name : "Sandra Velarde Gonzalez (ETNIC)"
+github:
+  repository: w3c/wai-fundamentals-overview
+  path: content/index.fr.md
+footer: > # Text in footer in HTML
+  <p><strong>Date :</strong> Mise à jour : 19 août 2020.</p>
+  <p><strong>Rédactrice :</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Développé par le Groupe de travail Éducation et Promotion (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+
+---
+
+Les ressources suivantes contiennent des introductions à l'accessibilité :
```

Comment:
The English sentence is:
> The following resources provide information for getting started with accessibility:

The translation should be closer to "for getting started", rather than "introductions" &mdash; partly because the first resource is called an introduction and the others are not.

Dear French speakers: Suggestions? Thanks!

Reply:
Suggestion:
Les ressources suivantes fournissent des informations pour démarrer en accessibilité:

What do you think @remibetin

Reply:
@vmmiler : Yes it's a good suggestion (with a space before the colon). Just a personal preference : "débuter" (more formal) rather than "démarrer" ?